From 35ddfd15b4742f73fd677ea6fb1a6bfa5fa709a9 Mon Sep 17 00:00:00 2001
From: Anthony Iliopoulos <ailiop@altatus.com>
Date: Mon, 15 Feb 2021 13:24:04 +0100
Subject: [PATCH] extentions makefile: add check to the phony targets

check-recurse from the top level makefile errors out when descenting
into the extentions directory, since the makefile there doesn't include
a check target.

Signed-off-by: Anthony Iliopoulos <ailiop@altatus.com>
---
 extensions/GNUmakefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git extensions/GNUmakefile.in extensions/GNUmakefile.in
index 956ccb3..78ce2ee 100644
--- a/extensions/GNUmakefile.in
+++ b/extensions/GNUmakefile.in
@@ -79,7 +79,7 @@ targets_install :=
 
 .SECONDARY:
 
-.PHONY: all install uninstall clean distclean FORCE
+.PHONY: all install uninstall check clean distclean FORCE
 
 all: ${targets}
 
-- 
2.30.1

